features/command-event.yml

```diff
@@ -0,0 +1,5 @@
+name: Command event
+description: The `command` event fires on an element that is controlled via a button with valid commandForElement and command values, whenever the button is interacted with (such as being clicked).
+spec: https://html.spec.whatwg.org/multipage/webappapis.html#handler-onchange
+compat_features:
+  - api.HTMLElement.command_event
```

Comment:
I think this is something a bit bigger, combining `html.elements.button.{command,commandFor}`, this event, and `api.HTMLButtonElement.{command,commandFor}`.

(Also, the MDN docs notwithstanding, I don't get it and would love to see the spec which I cannot find. It's a mystery!)

Reply:
This is actually a duplicate of https://github.com/web-platform-dx/web-features/pull/2411, and is part of "Invokers".

Reply:
ack, removed!